test/hw10/test_hw10_sip500.py

```diff
@@ -0,0 +1,128 @@
+"""TEST Homework 10.1."""
+import json
+import os
+from collections import namedtuple
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+from bs4 import BeautifulSoup
+
+from hw.hw10.hw10_sip500_2 import centrobank, code_name_price, companies
+from hw.hw10.hw10_sip500_2 import p_e, potential_profit, write_files, year_growth
+
+
+@pytest.fixture()
+def data_1_of_10_pages_companies():
+    with open("test/hw10/html1.html") as f:
+        return f.read()
+
+
+@pytest.fixture()
+def data_page_company():
+    with open("test/hw10/html2.html") as f:
+        return BeautifulSoup(f.read(), "html.parser")
+
+
+def xml_sberbank():
+    with open("test/hw10/asp.xml", "rb") as f:
+        return f.read()
+
+
+@pytest.fixture()
+def res():
+    return (
+        [{"1": 1}, {"2": 2}],
```

Comment:
It's unclear the meaning of this function from the name. Also this returned list can be generated programmatically